src/activity/events/components/EventDate.js

```diff
@@ -0,0 +1,18 @@
+import PropTypes from "prop-types";
+import "../events.css";
+
+function EventDate({ month, date }) {
+	return (
+		<div className="event-date">
+			<p id="month">{month}</p>
+			<p id="day">{date}</p>
+		</div>
+	);
+}
+
+export default EventDate;
+
+EventDate.propTypes = {
+	month: PropTypes.number.isRequired,
+	date: PropTypes.number.isRequired,
+};
```

Comment:
From the console:
Warning: Failed prop type: Invalid prop `month` and `date` of type `string` supplied to `EventDate`, expected `number`.

cause: fakedata stores the month and date as strings using`toLocaleString`

```suggestion
	month: PropTypes.string.isRequired,
	date: PropTypes.string.isRequired,```